styles/base/panorama_base.sass

```diff
@@ -34,10 +34,10 @@ TextTooltip
 	font-size: 12px
 	color: #111111ff
 
-    visibility: collapse
+	visibility: collapse
 
 TextTooltip.TooltipVisible
-    visibility: visible
+	visibility: visible
```

Comment:
Ditto

Reply:
Now I'm off mobile I can see that this is perfectly fine. GitHub sucks with diff rendering some times